.ci/baseline.benchmark

```diff
@@ -0,0 +1,15 @@
+Running benchmark 1/3...
```

Comment:
This is overengineering.
This file should have only one digit `8.56`
And probably no files , and just variable in shell file that defined our baseline value.

Reply:
Sure, I will make it a variable inside the script to avoid adding additional file type check suppression.

Reply:
done